src/lib/fixers.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable no-console */
+const { request } = require('@octokit/request');
+
+async function inviteUser({ username, organization, token, role = 'member', dryRun = false }) {
+  console.log(`Setting ${username} up as a(n) ${role} for ${organization}.`);
+  if (dryRun) return;
+  // Do the thing!
+  const result = await request('PUT /orgs/{org}/memberships/{username}', {
+    headers: {
+      authorization: `token ${token}`
+    },
+    org: organization,
+    username: username,
+    role: role
+  });
+  if (result.status === 200) {
+    console.log(`${username} successfully invited to ${organization}!`);
+  } else {
+    console.log(`ERROR ERROR! ${username} could not be invited to ${organization}!`);
+  }
+}
+
+async function removeMember({ username, organization, token, dryRun = false }) {
+  console.log(`Removing ${username} from ${organization}.`);
+  if (dryRun === true) return;
+  // Do the thing!
+  const result = await request('DELETE /orgs/{org}/members/{username}', {
+    headers: {
+      authorization: `token ${token}`
+    },
+    org: organization,
+    username: username
+  });
+  if (result.status === 204) {
+    console.log(`${username} successfully removed from ${organization}.`);
+  } else {
+    console.log(`ERROR ERROR! ${username} could not be removed from ${organization}!`);
```

Comment:
Another non-blocking thing, but something that should be done eventually is to have these error states set the right [exit code](https://nodejs.org/api/process.html#process_process_exitcode) so that if its chained on the command line it properly stops execution (could be here or elsewhere).

Reply:
(or just throw an error from here and let the promise rejection result in the failed exit code)

Reply:
Want to reiterate this as the correct flow for a node-based CLI.